### DISCUSSION TITLE
Support for old door and motion sensors?

### DISCUSSION THREAD

**Discussion by meierthomas** [SecKatie/ha-wyzeapi#396] [General]
I have successfully used the wyzesense HACS integration to integrate the motion and door sensors communicating via the USB base station. You had to add the sensor via the .scan service. This was on core-2021.9.

I'm building a new HA instance from the ground up and can't find wyzesense anymore but instead wyzeapi. Are the old sensors still supported? If not any idea how I can continue to use these old sensors?

**Comment by Kerdika** [comment]
Wyzesense still works fine. I set it up a couple months ago and it working great. I even have the usb dongle remote from the server with usbip running an a gen1 pi.
